src/components/Header/header.css

```diff
@@ -0,0 +1,16 @@
+.menu-button {
+  background-color: var(--primary-color);
+  padding: 15px;
```

Comment:
are you sure about sizes? Compare with design (and mobile)

Reply:
Need a call